### PR TITLE
BUG fix recipe for conda-smithy

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by beckermr** [comment]
CI here won't act normally. Does this change look ok to you all @conda-forge/staged-recipes ?

cc @xhochy for viz

**Comment by beckermr** [comment]
Alright, Let's try this. We can back it out of the downstream feedstock later, but I don't think we'll need to.

File: recipes/intel-repack/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% set version = "2019.5" %}
 {% set buildnum = "281" %}
 
+{% set my_channel_targets = channel_targets if channel_targets is defined else ['conda-forge', 'defaults'] %}
```
Comment (CJ-Wright) [inline recipes/intel-repack/meta.yaml]: I didn't realize that `is defined` was valid jinja2 syntax, awesome!

Comment (beckermr) [reply recipes/intel-repack/meta.yaml]: I need to file a smithy bug report.

Comment (jakirkham) [reply recipes/intel-repack/meta.yaml]: Can you please link the issue here once you file it?

Comment (beckermr) [reply recipes/intel-repack/meta.yaml]: Yeah for sure! The basic problem is that channel_targets is not defined for some of the recipe handling somewhere.

**Pull request merged**